### PR TITLE
Create StepDTO

### PR DESCRIPTION
**Pull request by Duskmage2009**
Привiт , глянь мiй ДТО будь ласка)

### CODE REVIEW

**Comment by bielik-b** [comment]
Привет, да всё норм, только утверждают наверное Артем или Тарас, я хз как
это делать😅

Вт, 7 февр. 2023 г. в 20:58, Kirill ***@***.***>:

**Review by VatBane — APPROVED** [review]
I'm not sure that we need default constructor, maybe we'll need to delete it later .

**Review by bielik-b — APPROVED** [review]

**Review by Serhx4 — APPROVED** [review]
Looks good to me. The same DTO pattern realization we had on previous lecture.

**Review by Serhx4 — APPROVED** [review]
@Duskmage2009, please, squash all commits into single one when you merge this request

File: src/main/java/executor/service/model/StepDTO.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	}
+
+	public String getValue() {
+		return value;
+	}
+
+	public void setValue(String value) {
+		this.value = value;
+	}
+
```
[the remark this answers is not in the document]
Comment (Duskmage2009) [reply src/main/java/executor/service/model/StepDTO.java:32]: Already done

**Review by artemkedyk — APPROVED** [review]

**Pull request merged**